turingschool-projects/grab_bag
# Fb refactor

#### What does  this PR do?
Refactor FacebookService and add methods to call 1000 uploaded photos and 1000 tagged photos

#### Where should the reviewer start?
Just app/services/facebook_service.rb

#### How should this be manually tested?
run rspec. There should only be a failing test saying int wants "25 photos" but you should see "xx photos" xx depending on the user_id and token you use

#### Questions:
  - Do Migrations Need to be ran? no
  - Do Environment Variables need to be set? no
